gen/gen.go

```diff
@@ -28,16 +30,31 @@ func validateDriver(s string) error {
 }
 
 var (
+	//go:embed services.tmpl
+	services_template_resource string
 	//go:embed atlas.tmpl
-	resource string
-	tmpl     *template.Template
+	main_template_resource string
+	tmpl                   *template.Template
 )
 
 func init() {
-	t, err := template.New("atlas.tmpl").Parse(resource)
+	// Based on: https://dev.to/moniquelive/passing-multiple-arguments-to-golang-templates-16h8
+	t := template.New("atlas-sync-action").Funcs(template.FuncMap{"args": func(els ...any) []any {
+		return els
+	}})
+
+	t, err := t.Parse(main_template_resource)
+
 	if err != nil {
-		panic(err)
+		log.Fatalf("Unable to load main template %v", err)
 	}
+
+	t, err = t.Parse(services_template_resource)
+
+	if err != nil {
+		log.Fatalf("Unable to load services template %v", err)
+	}
+
```

Comment:
Use template Must. See: https://pkg.go.dev/text/template#Must

Reply:
Cool, wasn't familiar with this!